### PR TITLE
zathura-pdf-mupdf: 0.3.3 -> 0.3.4

### PR DESCRIPTION
**Pull request by Profpatsch**
Support for mupdf 1.14.

### CODE REVIEW

**Comment by 7c6f434c** [comment]
@GrahamcOfBorg build zathura

**Comment by 7c6f434c** [comment]
Just in case, @cstrahan

File: pkgs/applications/misc/zathura/pdf-mupdf/default.nix [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-    sha256 = "1zbdqimav4wfgimpy3nfzl10qj7vyv23rdy2z5z7z93jwbp2rc2j";
+  # pwmt.org server was down at the time of last update
+  # src = fetchurl {
+  #   url = "https://pwmt.org/projects/zathura-pdf-mupdf/download/${name}.tar.xz";
+  #   sha256 = "1zbaqimav4wfgimpy3nfzl10qj7vyv23rdy2z5z7z93jwbp2rc2j";
+  # };
+  src = fetchFromGitHub {
+    owner = "pwmt";
+    repo = "zathura-pdf-mupdf";
+    rev = "0.3.4";
```
Comment (Synthetica9) [inline pkgs/applications/misc/zathura/pdf-mupdf/default.nix]:
```suggestion
    rev = version;
```

**Review by Synthetica9 — CHANGES_REQUESTED** [review]
See suggested change

**Comment by 7c6f434c** [comment]
@GrahamcOfBorg build zathura

File: pkgs/applications/misc/zathura/pdf-mupdf/default.nix (hunk 1 above)
Comment (Profpatsch) [reply pkgs/applications/misc/zathura/pdf-mupdf/default.nix]: Good catch!

**Comment by Profpatsch** [comment]
@GrahamcOfBorg build zathura

**Pull request merged**